docs/content/latest/deploy/multi-dc/2dc-deployment.md

```diff
@@ -111,3 +111,31 @@ Connect to “yugabyte-consumer” universe using the YSQL shell (`ysqlsh`) or t
 **For bidirectional replication**
 
 Repeat the steps above, but pump data into “yugabyte-consumer”. To avoid primary key conflict errors, keep the key space for the two universes separate.
+
+
+{{< note title="How to check replication lag" >}}
```

Comment:
```suggestion
{{< note title="Note" >}}
```
The Note title is only intended for Note, Important, Warning, and other admonitions.

Reply:
@stevebang  On the same page above it's a "tip" with a custom title `Recommended Reading`.

Reply:
OK, letting it slide for now — this use and others should be handled differently. But, I will have to propose an alternative.